xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd

```diff
@@ -956,7 +956,7 @@ Rail transport, Roads and Road transport
 			</xsd:element>
 			<xsd:element name="TactileActuators" type="xsd:boolean" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Whether Lift has a tactile actuator.</xsd:documentation>
+					<xsd:documentation>Whether there are buttons to start the TRAVELATOR if it is not moving, and whether these buttons are intended and suited for the visually impaired.</xsd:documentation>
```

Comment:
No, these here are the *TactileActuators* of the **LiftEquipment**. The change for Escalator and Travelator is already done (cf. line 726), I introduced it as a commit in #705.

Reply:
better with the new commit? @trurlurl